examples/example01.toit

```diff
@@ -1,13 +1,14 @@
+// Copyright (c) 2021 Ekorau LLC
 
-import ..src.test_util show *
+import btest show *
```

Comment:
This is cleaner but a bit more annoying:
on the one hand users can more easily copy the examples, on the other hand it introduces a package.yaml/lock file here, and requires the editing in an editor/LSP set up with `examples` as root.
(I use `code examples`).
If this is too annoying to you, leave this change out.